.infrastructure/cluster-ip.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: todoapp-clusterip
+  namespace: todoapp
+spec:
+  selector:
+    app: todolist
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 8080
+  type: ClusterIP
```

Comment:
![image](https://github.com/user-attachments/assets/69610b73-743c-473c-b924-14bf80c6b6a7)

Add Eof(End-of-file), need to add a blank line at the end

Reply:
fixed